README.rst

```diff
@@ -133,12 +139,18 @@ for a ``"source"`` key in the dictionary and, secondly, under the key ``0``.
 Command Line Arguments
 ~~~~~~~~~~~~~~~~~~~~~~
 
-The decorator can be used to pass command line arguments to ``julia``. See the
-following example.
+The decorator can be used to pass command line arguments to ``julia``. An important
+detail is that you need to differentiate between options passed to the Julia executable
+and arguments passed to the script.
```

Comment:
Add:

*Options and arguments must be list items; do not combine several options in a string."